src/Refactoring-UI/StRefactoringPreviewPresenter.class.st

```diff
@@ -90,7 +90,7 @@ RBRefactoringPreviewPresenter >> columns [
 ]
 
 { #category : #initialization }
-RBRefactoringPreviewPresenter >> connectPresenters [
+StRefactoringPreviewPresenter >> connectPresenters [
 
 	super connectPresenters.
```

Comment:
why calling super connectPresenters?

Reply:
I've removed it now, it makes no sense to have it here. Good catch!